### PR TITLE
Add badges

### PR DESCRIPTION
**Pull request by hzoo**
Check https://github.com/babel/babel/blob/hzoo-patch-1/README.md#packages

### CODE REVIEW

**Comment by danez** [comment]
Awesome work

File: README.md [hunk 1]
```diff
@@ -46,5 +46,205 @@ https://github.com/babel/babel/issues/2168
...
+| [`babel-traverse`](/packages/babel-traverse) | [![npm](https://img.shields.io/npm/v/babel-traverse.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-traverse) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-traverse)](https://david-dm.org/babel/babel?path=packages/babel-traverse) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-traverse)](https://david-dm.org/babel/babel?path=packages/babel-traverse#info=devDependencies) |
+| [`babel-generator`](/packages/babel-generator) | [![npm](https://img.shields.io/npm/v/babel-generator.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-generator) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-generator)](https://david-dm.org/babel/babel?path=packages/babel-generator) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-generator)](https://david-dm.org/babel/babel?path=packages/babel-generator#info=devDependencies) |
+
+`babel-core` is the Babel compiler itself; it exposes the `babel.transform` method, where `transformedCode = transform(src)`.
```
Comment (danez) [inline README.md]: transform returns an object `transformedCode = transform(src).code`.

File: README.md [hunk 2]
```diff
@@ -46,5 +46,205 @@ https://github.com/babel/babel/issues/2168
...
+
+The compiler can be broken down into 3 parts:
+- The parser: [`babylon`](https://github.com/babel/babylon) (moved to a seperate repo)
```
Comment (danez) [inline README.md]: maybe also add not that its versioned independently

File: README.md [hunk 3]
```diff
@@ -46,5 +46,205 @@ https://github.com/babel/babel/issues/2168
...
+
+#### Other
+
+| Package | Version | Dependencies | DevDependencies |
+|--------|-------|------------|----------|
+| [`babel-cli`](/packages/babel-cli) | [![npm](https://img.shields.io/npm/v/babel-cli.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-cli) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-cli)](https://david-dm.org/babel/babel?path=packages/babel-cli) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-cli)](https://david-dm.org/babel/babel?path=packages/babel-cli#info=devDependencies) |
+| [`babel-types`](/packages/babel-types) | [![npm](https://img.shields.io/npm/v/babel-types.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-types) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-types)](https://david-dm.org/babel/babel?path=packages/babel-types) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-types)](https://david-dm.org/babel/babel?path=packages/babel-types#info=devDependencies) |
```
Comment (danez) [inline README.md]: You mention it in the core section above that's why I'm not sure if this might be also considered core?

File: README.md [hunk 4]
```diff
@@ -46,5 +46,205 @@ https://github.com/babel/babel/issues/2168
...
+- [`babel-polyfill`](/packages/babel-polyfill) is [literally a wrapper](https://github.com/babel/babel/blob/master/packages/babel-polyfill/src/index.js) around [`core-js`](https://github.com/zloirock/core-js) and [regenerator-runtime](https://github.com/facebook/regenerator/tree/master/packages/regenerator-runtime). Check out the [docs](https://babeljs.io/docs/usage/polyfill/).
+- [`babel-runtime`](/packages/babel-runtime) is similar to the polyfill except that it doesn't modify the global scope and is to be used with [`babel-plugin-transform-runtime`](/packages/babel-plugin-transform-runtime) (usually in library/plugin code). Check out the [docs](https://babeljs.io/docs/plugins/transform-runtime/)
+- [`babel-register`](/packages/babel-register) is a way to automatically compile files with babel on the fly by binding to node's require. Check out the [docs](http://babeljs.io/docs/usage/require/)
+- [`babel-template`](/packages/babel-template) is a helper function to make AST nodes. Instead you can pass a string representing the code you want to create rather than tediously building them using `babel-types.
```
Comment (danez) [inline README.md]: missing `

File: README.md [hunk 5]
```diff
@@ -46,5 +46,205 @@ https://github.com/babel/babel/issues/2168
...
+- [`babel-helpers`](/packages/babel-helpers)) is a set of premade `babel-template` functions that are used in some babel plugins.
```
Comment (danez) [inline README.md]: There is `)` after the link

File: README.md (hunk 3 above)
Comment (hzoo) [inline README.md]: Yeah I wasn't sure - I wanted it to be just the parser,generator,traverse so I moved it below (forgot to remove it above)

Comment (hzoo) [inline README.md]: I should post my slides from my talk somewhere

**Pull request merged**